python/ray/_private/accelerators/accelerator.py

```diff
@@ -1,6 +1,16 @@
 from abc import ABC, abstractmethod
 from typing import Dict, List, Optional, Tuple
 
+# https://github.com/ray-project/ray/issues/54868
+# It aims to avoid overriding the accelerator ids environment variables when
+# the number of accelerators is zero.
+# For example, when this environment variable is set, if a user sets `num_gpus=0`
+# in the `ray.init()` call, the environment variable `CUDA_VISIBLE_DEVICES` will
+# not be set to an empty string.
+RAY_EXPERIMENTAL_NO_ACCEL_OVERRIDE_ON_ZERO_ENV_VAR = (
```

Comment:
Lets remove the `_EXPERIMENTAL_` part. We don't use it for feature flags.

```
RAY_ACCEL_ENV_VAR_OVERRIDE_ON_ZERO = env_bool("RAY_ACCEL_ENV_VAR_OVERRIDE_ON_ZERO", True)
```

Reply:
done